README.md

```diff
@@ -1,18 +1,37 @@
+# ESP32Marauder for ESP32-2432S024R with USB-C 🚀✨
 
-# ESP32Marauder for ESP32-2432S024R with USB-C.
-ESP32 with 2.4inch 240*320 LCD TFT Module and Touch
-![ESP32-2432S024R](https://github.com/b00mekk/ESP32-Marauder-ESP32-2432S024R/blob/main/ESP32-Arduino-LVGL-WIFI-Bluetooth-Development-Board-2-4-240-320-Smart-Display-Screen-2-4inch.jpg_.png?raw=true)
+ESP32 with 2.4-inch 240x320 LCD TFT Module and Touch 📺👆
 
-# Tested on:
-- [Module 1]([https://aliexpress.com/item/1005006969878456.html](https://s.click.aliexpress.com/e/_DD5uQKp))
+![ESP32-2432S024R](https://github.com/Asmodeux2/ESP32Marauder_ESP32-2432S024R_CYD/blob/main/marauder.jpg)
 
-# How to setup
-1. Change TFT_eSPI/User_Setup.h with [this one](https://github.com/b00mekk/ESP32-Marauder-ESP32-2432S024R/blob/main/User_Setup.h)
-2. Download [esp32_marauder](https://github.com/b00mekk/ESP32-Marauder-ESP32-2432S024R/tree/main/esp32_marauder) files
-3. Upload to ESP32-2432S024R using USB-C
+## Tested on:
+- [Module 1](https://s.click.aliexpress.com/e/_DD5uQKp) 🛒🌟
 
-# Pre-build binaries
-1. visit the flasher page https://esp.huhn.me/
-2. Flash the binaries included in the **esp32_marauder\build\esp32.esp32.d32_pro/**  folder in this order 1. partitions, 2.bootloader. 3. bins
- example
-![immagine](https://github.com/user-attachments/assets/b1dead25-7606-485b-9831-5a0921b3bc0e)
+## How to Setup
+
+1. **Update TFT_eSPI/User_Setup.h**:
+   - Replace the contents of `TFT_eSPI/User_Setup.h` with the [updated version](https://github.com/b00mekk/ESP32-Marauder-ESP32-2432S024R/blob/main/User_Setup.h). 📄🔄
+
+2. **Download Files**:
+   - Download the [esp32_marauder](https://github.com/b00mekk/ESP32-Marauder-ESP32-2432S024R/tree/main/esp32_marauder) files. 📁💾
+
+3. **Upload to ESP32-2432S024R**:
+   - Use USB-C to upload the files to your ESP32-2432S024R. 🔌💻
+
+## Pre-built Binaries
+
+1. **Visit the Flasher Page**:
```

Comment:
I suggest making step one like "download the bins from https://github.com/Asmodeux2/ESP32Marauder_ESP32-2432S024R_CYD/tree/main/esp32_marauder/build/esp32.esp32.d32_pro "     
Then make the "visit the flasher website" the 2nd step.

Reply:
Yes, that's a good idea, but I just arranged for another author to add this firmware to their repository in WebFlasher, specifically https://github.com/Fr4nkFletcher/ESP32-Marauder-Cheap-Yellow-Display.